site/content/en/docs/Installation/Creating Cluster/eks.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Amazon Elastic Kubernetes Service"
+linkTitle: "Amazon Web Services"
+weight: 20
+description: >
+  Setting up an Amazon Web Services EKS cluster
```

Comment:
Can you make the descriptions consistent between the different pages?

Reply:
Done!